catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/ReplaceExistingProjectDialogFragment.java

```diff
@@ -78,10 +78,12 @@ public void onCheckedChanged(RadioGroup group, @IdRes int checkedId) {
 				}
 			}
 		});
+		radioGroup.check(R.id.rename);
```

Comment:
The default selection for the `RadioGroup` can be changed via the layout resource [here](https://github.com/Catrobat/Catroid/blob/f1f8cfe6c80149bdae9f2244fad8cc5a0ccdd7cb/catroid/src/main/res/layout/dialog_overwrite_project.xml#L39). You can also rearrange the sequence (if you want rename before replace) and if you want to change the default visibility of the `inputLayout` there as well.